linux/utils/enable_vmtools_logging.yml

```diff
@@ -72,6 +72,13 @@
   ansible.builtin.debug: var=vmtools_config
   when: enable_debug
 
+- name: "Collect VMware Tools config file to local log directory"
+  include_tasks: fetch_file.yml
+  vars:
+    fetch_file_src_path: "{{ vmtools_config_file }}"
+    fetch_file_dst_path: "{{ current_test_log_folder }}/"
```

Comment:
no need "/"

Reply:
Using '/' is to set 'flat=true' for linux fetching file.